doc/presentación/defensa.tex

```diff
@@ -0,0 +1,313 @@
+        Una red bayesiana $\mathfrak{B} = \lbrace \mathcal{G}, \mathbb{P} \rbrace$ está definida por:
+        \begin{itemize}
+            \item $\mathcal{G}=(V,E)$ 
+            \item $(\Omega, \mathbb{P})$.
+            \item $\textbf{V}=V[i], i=1,...,N$, $(\Omega, \mathbb{P})$ 
+            de tal manera que $\mathbb{P}(V[1],...,V[N])= \prod_{i=1}^{n}\mathbb{P}(V[i]|pa(V[i]))$
+        \end{itemize}
+    \end{definition}     
+\end{frame}
+
+\begin{frame}{Redes bayesianas}
+	\begin{center}
+		Ejemplo de grafo dirigido acíclico \\[12pt]
+		\includegraphics[width=0.35\textwidth,keepaspectratio]{dag.png}
+		\\
+		\footnotesize(Una red bayesiana consiste en un modelo estructural y un conjunto de probabilidades)
+    \end{center}
+\end{frame}
+
+\begin{frame}{Entropía}
+	\begin{definition}[Entropía] \label{def:entropy}
+        Para $X$ variable aleatoria discreta:  
+        
+        $$H(X):= - \sum_{x} P(x)\log[P(x)]$$
+        
+        donde $X$ toma valores en $\mathcal{X}$, $P:\mathcal{X} \rightarrow [0,1]$.
+        La entropía conjunta de las variables $X_1...X_N$ se define por 
+        $$H(X_1,...,X_N):=-\sum_{x_1}...\sum_{x_N}P(x_1,...,x_N)\log[P(x_1,...,x_N)]$$
+    \end{definition}
+\end{frame}
+
+%%%%
+\section{Contribuciones}
+
+%%
+\subsection{Resultados}
+
+\begin{frame}{Redes de pases}
+%   	\begin{alertblock}{Attention}
+%   		\textlatin{This is an important alert}
+%   	\end{alertblock}
+    \begin{columns}[t]
+        \column{.5\textwidth}
+        \includegraphics[width=\columnwidth,height=3cm]{plot_england.png}
+        \captionof{figure}{Entropía conjunta de Inglaterra}
+        \includegraphics[width=\columnwidth,height=3cm]{plot_england_simpl.png}
+        \captionof{figure}{Entropía conjunta simplificada de Inglaterra}
+        \column{.5\textwidth}
+        \includegraphics[width=\columnwidth,height=3cm]{plot_norw.png}
+        \captionof{figure}{Entropía conjunta de Noruega}
+        \includegraphics[width=\columnwidth,height=3cm]{plot_norw_simpl.png}
+        \captionof{figure}{Entropía conjunta simplificada de Noruega}
+    \end{columns} 
+\end{frame}
+
+\begin{frame}{Partido Inglaterra - Noruega}
+	%   	\begin{alertblock}{Attention}
+	%   		\textlatin{This is an important alert}
+	%   	\end{alertblock}
+		\begin{columns}[t]
+			\column{.5\textwidth}
+			\includegraphics[width=\columnwidth,height=3cm]{entropy_match_eng.png}
+			\captionof{figure}{Entropía conjunta de Inglaterra}
+			\includegraphics[width=\columnwidth,height=3cm]{entropy_match_engl_simpl.png}
+			\captionof{figure}{Entropía conjunta simplificada de Inglaterra}
+			\column{.5\textwidth}
+			\includegraphics[width=\columnwidth,height=3cm]{entropy_match_norw.png}
+			\captionof{figure}{Entropía conjunta de Noruega}
+			\includegraphics[width=\columnwidth,height=3cm]{entropy_match_norw_simpl.png}
+			\captionof{figure}{Entropía conjunta simplificada de Noruega}
+		\end{columns} 
+	\end{frame}
+
+\begin{frame}{Comparativa por jugadoras}
```

Comment:
Esto es mejor que lo hagas en dos transpas diferentes. Una de las preguntas es si puede ayudar la entropía a entender el cambio, puede que sí.
Lo que tendrías también que hader es ponerlas con algún orden, si puede ser el mismo. O alfabético o por orden de entropía, pero así no se be nada y es muy difícil comparar.
De hecho, podías añadir transparencias que compararan el desempeño de Inglaterra general y en el partido, y el de Noruega general y en el partido. 4 transparencias en total.

Reply:
de las tablitas con la entropía de cada jugadora te refieres?